src/components/campaigns/grid/CampaignGrid.tsx

```diff
@@ -119,53 +129,97 @@ export default function CampaignGrid() {
       renderCell: (params: GridRenderCellParams) => {
         return <DisplayCampaignType params={params} />
       },
+      align: 'left',
       width: 250,
     },
     {
       field: 'description',
       headerName: t('campaigns:description'),
-      align: 'right',
       ...commonProps,
+      align: 'left',
       width: 350,
     },
+    {
+      field: 'reachedAmound',
+      headerName: t('campaigns:amount'),
+      ...commonProps,
+      align: 'right',
+      width: 200,
+      renderCell: (cellValues: GridRenderCellParams) => (
+        <ExternalLink href={`/admin/donations/campaign/${cellValues.row.id}`}>
+          <DisplayDonationAmount params={cellValues} />
+        </ExternalLink>
+      ),
+    },
     {
       field: 'targetAmount',
       headerName: t('campaigns:targetAmount'),
-      align: 'right',
       ...commonProps,
+      align: 'right',
+      width: 150,
+      renderCell: (cellValues: GridRenderCellParams) => <>{money(cellValues.row.targetAmount)}</>,
     },
     {
       field: 'currency',
       headerName: t('campaigns:currency'),
       ...commonProps,
+      align: 'left',
     },
     {
       field: 'startDate',
       headerName: t('campaigns:startDate'),
       align: 'left',
       width: 230,
       headerAlign: 'left',
+      renderCell: (cellValues: GridRenderCellParams) => (
+        <>
+          {formatRelative(parseISO(cellValues.row.startDate), new Date(), {
+            locale: i18n.language == 'bg' ? bg : enUS,
+          })}
+        </>
+      ),
     },
     {
       field: 'endDate',
       headerName: t('campaigns:endDate'),
       align: 'left',
       width: 230,
       headerAlign: 'left',
+      renderCell: (cellValues: GridRenderCellParams) => (
+        <>
+          {formatRelative(parseISO(cellValues.row.endDate), new Date(), {
+            locale: i18n.language == 'bg' ? bg : enUS,
+          })}
+        </>
+      ),
     },
     {
       field: 'createdAt',
       headerName: t('campaigns:createDate'),
       align: 'left',
       width: 230,
       headerAlign: 'left',
+      renderCell: (cellValues: GridRenderCellParams) => (
+        <>
+          {formatRelative(parseISO(cellValues.row.createdAt), new Date(), {
+            locale: i18n.language == 'bg' ? bg : enUS,
+          })}
+        </>
+      ),
     },
     {
       field: 'updatedAt',
       headerName: t('campaigns:updatedAt'),
       align: 'left',
       width: 230,
       headerAlign: 'left',
+      renderCell: (cellValues: GridRenderCellParams) => (
+        <>
+          {formatRelative(parseISO(cellValues.row.updatedAt), new Date(), {
```

Comment:
I think you can use `getRelativeDate` as well. It wraps the format relative with a `new date`

You may pass the locale as second param to `getRelativeDate` after adding it as optional to the fn